p5/sketch/Skia2DRenderer/base.py

```diff
@@ -108,11 +108,8 @@ def skia_surface(self, window=None, size=None):
 
     # create a new surface everytime
     def create_surface(self, size=None):
-        if not size:
-            size = self._size
-        self._size = size
```

Comment:
I think self._size still needs to be updated in the new version for this to work:
```
    @property
    def size(self):
        return self._size
```